src/cp/cp.js

```diff
@@ -1,6 +1,27 @@
+import { fileURLToPath } from 'node:url';
+import path, { dirname } from 'node:path';
+import childProcess from 'node:child_process';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const scriptPath = path.join(__dirname, 'files', 'script.js');
+
 const spawnChildProcess = async (args) => {
-    // Write your code here
+  const child = childProcess.spawn('node', [scriptPath, ...args]);
```

Comment:
Maybe it is a better idea to use `.fork`.

Reply:
As I remember .fork is sync operation and i thought it's prohibited